ONSdigital/eq-author
# Min Value Validation

### What is the context of this PR?
Implements MinValue validation on numeric answers

### How to review 
- Run tests
- Ensure feature works as per https://trello.com/c/PfYRU46S/404-validation-min-value-front-end-changes

### Dependencies

API: https://github.com/ONSdigital/eq-author-api/pull/106
Schema: https://github.com/ONSdigital/eq-author-graphql-schema/pull/55
